---
layout: archive
title: "Resources"
permalink: /resources/
author_profile: true
---

### Integration between single-cell genomics and spatial-omics
- **MaxFuse** [`github.com/shuxiaoc/maxfuse`](https://github.com/shuxiaoc/maxfuse): Matching across weakly-linked modalities.
- **MARIO** [`github.com/shuxiaoc/mario-py`](https://github.com/shuxiaoc/mario-py): Matching across partial strongly-linked modalities.

### Information fusion between expression and image-level information
- **CellLENS** [`github.com/sggao/celllens`](https://github.com/sggao/celllens): Expression and image fusion for better cell type delineation.

### Quantitative analysis of tissue disease states
- **MESA** [`mesa-py.readthedocs.io`](https://mesa-py.readthedocs.io/en/latest/): Multiomics and Ecological Spatial Analysis.

### Spatial-omics and microbiome
- **MicroCart-analysis** [`github.com/BokaiZhu/microcart_analysis`](https://github.com/BokaiZhu/microcart_analysis): Microbiome Cartography (Part 1).
- **MicroCart-probe** [`github.com/BokaiZhu/microbiomeFISH`](https://github.com/BokaiZhu/microbiomeFISH): Microbiome Cartography (Part 2).

### Signal spillover correction in spatial-omics
- **REDSEA** [`github.com/nolanlab/REDSEA`](https://github.com/nolanlab/REDSEA): REinforcement Dynamic Spillover EliminAtion.
